README.md

```diff
@@ -313,6 +316,15 @@ tox -c tests/integration/
 
 This should run the integration tests and watch them run in a Firefox instance you can watch and interact with.
 
+#### Integration Test options
+
+- `TOX_ENV`: [Tox](https://tox.readthedocs.io/en/latest/config.html#tox) commandline variables.
+- `PYTEST_ENV`: [Pytest](https://docs.pytest.org/en/6.2.x/usage.html#) commandline variables.
```

Comment:
Q: Should these be `TOX_ENV` or `TOX_ARGS` (to match L157+L160 in the Makefile above)?
(same w/ `PYTEST_ENV` vs `PYTEST_ARGS`)

Reply:
Ah, good catch!